.env

```diff
@@ -0,0 +1,22 @@
+# MongoDB
+MONGO_PORT=27017
+MONGO_INITDB_ROOT_USERNAME=admin
+MONGO_INITDB_ROOT_PASSWORD=password
+
+## Hardened MongoDB
+MONGO_INITDB_DATABASE=nlpsandbox
+MONGO_USERNAME=nlpmongo
+MONGO_PASSWORD=nlpmongo
+
+# API SERVER
+SERVER_PROTOCOL=http://
+SERVER_DOMAIN=localhost
+SERVER_PORT=8080
+
+## DB
+DB_PROTOCOL=mongodb://
+DB_DOMAIN=localhost
+DB_PORT=27017
+DB_DATABASE=nlpsandbox
```

Comment:
DB_DATABASE, DB_USERNAME and DB_PASSWORD should take the same value as MONGO_INITDB_DATABASE, MONGO_USERNAME and MONGO_PASSWORD respectively.